[cob_obstacle_distance] does not build with cmake_build_type=Debug

OS: Ubuntu 20.04
ROS: noetic

linker error is:
```
/usr/bin/ld: CMakeFiles/cob_obstacle_distance.dir/src/distance_manager.cpp.o: undefined reference to symbol 'ccd_vec3_origin'
/usr/bin/ld: /lib/x86_64-linux-gnu/libccd.so.2: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
```

--- a/cob_obstacle_distance/CMakeLists.txt
+++ b/cob_obstacle_distance/CMakeLists.txt
@@ -12,14 +12,14 @@ set(EIGEN3_INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR})
 add_definitions(${EIGEN3_DEFINITIONS})
 
 find_package(orocos_kdl REQUIRED)
-
 find_package(PkgConfig REQUIRED)
 
 pkg_check_modules(ASSIMP assimp)
 
-pkg_check_modules(LIBFCL REQUIRED fcl)
-find_library(LIBFCL_LIBRARIES_FULL ${LIBFCL_LIBRARIES} ${LIBFCL_LIBRARY_DIRS})
-set(fcl_LIBRARIES "${LIBFCL_LIBRARIES_FULL}")
+find_package(fcl REQUIRED)
+find_package(ccd QUIET)
+set(CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMakeModules")
+include_directories(${fcl_INCLUDE_DIRS})
 
 catkin_package(
   CATKIN_DEPENDS cob_control_msgs cob_srvs dynamic_reconfigure eigen_conversions geometry_msgs kdl_conversions kdl_parser moveit_msgs roscpp roslib sensor_msgs shape_msgs std_msgs tf tf_conversions urdf visualization_msgs
@@ -29,19 +29,19 @@ catkin_package(
 )
 
 ### BUILD ###
-include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${FCL_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS} ${ASSIMP_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS} ${Boost_INCLUDE_DIRS} ${CCD_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${fcl_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS} ${ASSIMP_INCLUDE_DIRS})
 
 add_library(parsers src/parsers/mesh_parser.cpp src/parsers/stl_parser.cpp)
 add_dependencies(parsers ${catkin_EXPORTED_TARGETS})
-target_link_libraries(parsers assimp ${fcl_LIBRARIES} ${catkin_LIBRARIES})
+target_link_libraries(parsers assimp fcl ${catkin_LIBRARIES})
 
 add_library(marker_shapes_management  src/link_to_collision.cpp src/marker_shapes/marker_shapes_impl.cpp src/marker_shapes/marker_shapes_interface.cpp src/shapes_manager.cpp)
 add_dependencies(marker_shapes_management ${catkin_EXPORTED_TARGETS})
-target_link_libraries(marker_shapes_management parsers ${fcl_LIBRARIES} ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
+target_link_libraries(marker_shapes_management parsers fcl ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
 
 add_executable(${PROJECT_NAME} src/chainfk_solvers/advanced_chainfksolver_recursive.cpp src/${PROJECT_NAME}.cpp src/distance_manager.cpp src/helpers/helper_functions.cpp)
 add_dependencies(${PROJECT_NAME} ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
-target_link_libraries(${PROJECT_NAME} parsers marker_shapes_management ${fcl_LIBRARIES} ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
+target_link_libraries(${PROJECT_NAME} parsers marker_shapes_management fcl ${CCD_LIBRARIES} ${catkin_LIBRARIES} ${orocos_kdl_LIBRARIES})
 
 add_executable(debug_obstacle_distance_node src/debug/debug_obstacle_distance_node.cpp)
 add_dependencies(debug_obstacle_distance_node ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})

--- a/cob_obstacle_distance/package.xml
+++ b/cob_obstacle_distance/package.xml
@@ -21,15 +21,14 @@
   <depend>dynamic_reconfigure</depend>
   <depend>eigen_conversions</depend>
   <depend>eigen</depend>
-  <depend condition="$ROS_DISTRO != noetic">libfcl-dev</depend>
-  <depend condition="$ROS_DISTRO == noetic">fcl</depend>
+  <depend>fcl</depend>
   <depend>geometry_msgs</depend>
   <depend>kdl_conversions</depend>
   <depend>kdl_parser</depend>
+  <depend>libccd-dev</depend>
+  <depend>libfcl-dev</depend>
+  <depend>liborocos-kdl-dev</depend>
   <depend>moveit_msgs</depend>
-  <depend condition="$ROS_DISTRO == melodic">orocos_kdl</depend>
-  <depend condition="$ROS_DISTRO == noetic">liborocos-kdl-dev</depend>
-  <depend condition="$ROS_DISTRO == noetic">liborocos-kdl</depend>
   <depend>pkg-config</depend>
   <depend>roscpp</depend>
   <depend>roslib</depend>

